activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeVariableMappingIT.java

```diff
@@ -148,6 +150,111 @@ public void processTaskVarMapping() {
         processBaseRuntime.delete(processInstance.getId());
     }
 
+    @Test
+    public void processTaskVarMappingForMultiInstance() {
```

Comment:
maybe we can update the test names of the whole class to the agreed standard:
`should_somethingHappen_when_somethingElseHappens'
it'll be clearer that way what we are trying to do with each case

Reply:
ignore the long method warning: https://app.codacy.com/gh/Activiti/Activiti/pullRequest?prid=5752335

Reply:
Updated the names for tests

Reply:
Avoid using camel-case after a hyphen/underscore:
`should_NotHaveTaskVariables_when_MappingIsEmpty` 
should be
`should_notHaveTaskVariables_when_mappingIsEmpty`